pkg/csi/service/vanilla/controller.go

```diff
@@ -383,75 +383,178 @@ func (c *controller) ReloadConfiguration() error {
 	if err != nil {
 		return logger.LogNewErrorf(log, "failed to read config. Error: %+v", err)
 	}
-	newVCConfig, err := cnsvsphere.GetVirtualCenterConfig(ctx, cfg)
-	if err != nil {
-		log.Errorf("failed to get VirtualCenterConfig. err=%v", err)
-		return err
-	}
-	if newVCConfig != nil {
-		var vcenter *cnsvsphere.VirtualCenter
-		if c.manager.VcenterConfig.Host != newVCConfig.Host ||
-			c.manager.VcenterConfig.Username != newVCConfig.Username ||
-			c.manager.VcenterConfig.Password != newVCConfig.Password {
-
-			// Verify if new configuration has valid credentials by connecting to
-			// vCenter. Proceed only if the connection succeeds, else return error.
-			newVC := &cnsvsphere.VirtualCenter{Config: newVCConfig}
-			if err = newVC.Connect(ctx); err != nil {
-				return logger.LogNewErrorf(log, "failed to connect to VirtualCenter host: %q, Err: %+v",
-					newVCConfig.Host, err)
+	if multivCenterCSITopologyEnabled {
```

Comment:
Can this flag be enabled even on a single VC deployment ?

Reply:
yes. after `multi-vcenter-csi-topology` is released, user can still use single VC deployment.

Reply:
Got it, thanks!